.eslintrc-client-test

```diff
@@ -0,0 +1,7 @@
+---
+extends:
+  - "defaults/configurations/walmart/es6-test"
+  - "defaults/configurations/walmart/es6-react"
```

Comment:
This was a problem: I wanted the order to be `es6-react` then `es6-test`, but `es6-test` somehow override unrelated rules added by `es6-react` and I got a lint failure ( `no-extra-parens`). Does `extend` not do a "deep" extend?